### ISSUE TITLE
Feedback for Chardale: AI player

### ISSUE THREAD

**Issue by halfghaninne** [ioscowboi/rockpaperscissors#3]
CharDale, your AI player is formatted very well in its output -- I liked how the AI player's move returned as /////hidden/////, for example -- but on the backend I think there are some things that are not doing you favors.

Just to check: is it possible for two players -- neither of them a computer -- to play the game, using a Driver object? [These lines](https://github.com/ioscowboi/rockpaperscissors/blob/master/driver.rb#L29-L30") in driver.rb make it seem like that might not be possible; it looks like your AI features are hard-coded. 

I think getting that Player class working and pulling player names and scores from that file may free you up to write in an AI player method within it, so users have the option.

**Comment by ioscowboi** [comment]
You're correct, it is currently hard coded. I had it set for two players, but I changed it to speed up testing. I like your idea to allow the user to decide whether to play one or two player mode. I'll create a mode method for this.
